src/views/list_editable.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createNodeOrElementView } from './creators'
+import { EditableBlock } from './editable_block'
+import { ListCallback, ListView } from './list'
+
+export default createNodeOrElementView(
+  EditableBlock(ListView),
+  ListView.getElementType(),
+  ListCallback
+)
```

Comment:
No need for this.

Reply:
When I removed this file the list became uneditable, so do you mean using EditableBlock method inside the list.ts view file.

Reply:
Right.